docs/Setup-TRSS.md

```diff
@@ -0,0 +1,79 @@
+# TRSS Setup
+
+This page will document the setup of the TRSS service running at [trss.adoptium.net](https://trss.adoptium.net).
+
+## Setup
+
+#### Prerequisites:
+- Install [docker and docker-compose-plugin](https://docs.docker.com/compose/install/linux/)
+- Install `npm`
+- Install [`nginx`](https://www.nginx.com/resources/wiki/start/topics/tutorials/install/)
+
+#### Pre setup recommendations:
+- Dedicate a separate disk for `/var/lib/docker` (>30G) and one for the directory in which you kick off the TRSS service (>60G), the latter will be where TRSS stores its data.
+- Run the TRSS service with a non root user.
+
+### Before running the service:
+
+#### Connecting to Jenkins
+
+- Before you start the service, create a file named `trssConf.json`. This file will be stored in `aqa-test-tools/TestResultSummaryService/` once you clone the [aqa-test-tools](https://github.com/adoptium/aqa-test-tools) repository.
+- This file will contain the credentials needed by the TRSS service to connect to your Jenkins instance. It should look like this:
+```
+{
+	"https://yourjenkinsserver.com": {
+		"user" : "abc@example.com",
+		"password" : "123"   <=== the value can be token
+	}
+}
+```
+
+#### Nginx config
+
+A basic `/etc/nginx/nginx.conf` should include 
+```
+include /etc/nginx/sites-enabled/*;
+```
+which should include a symlink to `/etc/nginx/sites-enabled/defaults`.
+
+The file `/etc/nginx/sites-enabled/defaults` should contain a code block which passes all inbound traffic to `http://localhost:4000`. It should look something like this:
+```
+    location / {
+                proxy_pass http://localhost:4000;
+    }
+```
+
+`http://localhost:4000` runs another `nginx` service in a docker container. Its purpose is to redirect traffic to the appropriate TRSS docker container.
```

Comment:
Maybe mention that this docker container is something that will be created in a later step (along with various other containers for running TRSS), since it's not obvious at this stage if you're just reading through the doc.